modules/UI/videolayout/SmallVideo.js

```diff
@@ -463,8 +463,10 @@ export default class SmallVideo {
      * or <tt>DISPLAY_BLACKNESS_WITH_NAME</tt>.
      */
     selectDisplayMode(input) {
-        // Display name is always and only displayed when user is on the stage
-        if (input.isCurrentlyOnLargeVideo && !input.tileViewEnabled) {
+        if (!input.tileViewEnabled && input.isScreenSharing) {
```

Comment:
I did this because it never made sense to me to show the screen share in the thumbnail. So now we'd be consistent with mobile, FWIW.

Now, there is an interesting case: tile view. We switch away from tile view when someone starts sharing content, but they can force tile view. What should be displayed there? I'd say avatar + screen-share indicator, but I'd like confirmation on this.